metron-interface/metron-alerts/src/app/alerts/alerts-list/tree-view/tree-view.component.spec.ts

```diff
@@ -51,10 +52,12 @@ describe('TreeViewComponent', () => {
       providers: [
         SearchService,
         UpdateService,
-        GlobalConfigService,
         MetaAlertService,
         DialogService,
-        { provide: AppConfigService, useClass: FakeAppConfigService }
+        { provide: AppConfigService, useClass: FakeAppConfigService },
+        { provide: GlobalConfigService, useValue: {
```

Comment:
Add a stub here for the same reason mentioned in this comment: https://github.com/apache/metron/pull/1477/files#r314338346